ams/server/src/main/java/com/netease/arctic/server/dashboard/model/OptimizerInstanceInfo.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.server.dashboard.model;
+
+import com.netease.arctic.server.resource.OptimizerInstance;
+
+public class OptimizerInstanceInfo extends OptimizerInstance {
```

Comment:
The "OptimizerInstanceInfo" is just a DTO for AMS, so I don't think it should extend "Resouce". Also there are some duplicate params

Reply:
There are 9 parameters in there, and we can add a generator class to make it clear to add the field:
```
    List<OptimizerInfo> result =
        optimizerList.stream()
            .map(
                e ->
                    OptimizerInfo.builder()
                        .token(e.getToken())
                        .startTime(e.getStartTime())
                        .touchTime(e.getTouchTime())
                        .jobId(e.getResourceId())
                        .optimizerGroup(e.getGroupName())
                        .coreNumber(e.getThreadCount())
                        .memory(e.getMemoryMb())
                        .jobStatus("RUNNING")
                        .container(e.getContainerName())
                        .build())
            .collect(Collectors.toList());
```

Reply:
Yes, we may do better to make the DTO much simpler.